libprio.i

```diff
@@ -41,7 +41,7 @@
     }
 
     // Create a temporary stack variable for allocating a new opaque pointer
-    %typemap(in) T* (void *tmp) {
+    %typemap(in,numinputs=0) T* (void *tmp) {
```

Comment:
This removes the hanging parameters from the `KeyPair_new` function which previously required passing in a dummy value.